airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -61,6 +62,26 @@ class TITargetStatePayload(BaseModel):
     state: IntermediateTIState
 
 
+class TIDeferredStatePayload(BaseModel):
+    """Schema for updating TaskInstance to a deferred state."""
+
+    state: Annotated[
+        Literal[IntermediateTIState.DEFERRED],
+        # Specify a default in the schema, but not in code, so Pydantic marks it as required.
+        WithJsonSchema(
+            {
+                "type": "string",
+                "enum": [IntermediateTIState.DEFERRED],
+                "default": IntermediateTIState.DEFERRED,
+            }
+        ),
+    ]
+
+    kwargs: dict[str, Any]
```

Comment:
Should we call this `trigger_kwargs` or `next_kwargs` instead so it is clear?

Reply:
Also add default_factory as type so we don't need to accept empty kwargs
```suggestion
    kwargs: Annotated[dict[str, Any], Field(default_factory=dict)]
```

Reply:
Sure, renamed to `trigger_kwargs` and handled it